magefiles/helm.go

```diff
@@ -145,9 +147,13 @@ func updateChartValue(key, value string) {
 	}
 
 	//// write to file
-	newValueFile, err := yaml.Marshal(&n)
+	var b bytes.Buffer
+	yamlEncoder := yaml.NewEncoder(&b)
+	yamlEncoder.SetIndent(2)
+	yamlEncoder.Encode(&n)
```

Comment:
This line is repeated.
This causes the content to be written twice

Reply:
Oops, sorry. Good catch. Fixed